styles/alert.css

```diff
@@ -0,0 +1,47 @@
+@import "./global.css";
+
+.alert {
+  width: 100%;
+  padding: var(--spacing-md) var(--spacing-md);
+  background-color: var(--primary-color);
+  color: #fff;
+  border-radius: 4px;
+  font-size: 0.8rem;
+}
+
+.alert-info {
+  background-color: var(--info-color);
+}
+.alert-danger {
+  background-color: var(--danger-color);
+}
+.alert-success {
+  background-color: var(--success-color);
+}
+
+.alert-warning {
+  background-color: var(--warning-color);
+}
+
+.outline-danger {
+  background-color: #fff;
+  border: 1px solid var(--danger-color);
+  color: var(--danger-color);
+}
+
+.outline-info {
+  background-color: #fff;
+  border: 1px solid var(--info-color);
+  color: var(--info-color);
+}
+
+.outline-success {
+  background-color: #fff;
+  border: 1px solid var(--success-color);
+  color: var(--success-color);
+}
+.outline-warning {
+  background-color: #fff;
```

Comment:
instead of repeating color each time, you could this as a variable.

Reply:
thanks, Rahul will definitely add this.

Reply:
Just added the white-color varible